#!/usr/bin/env python
PACKAGE = "jaco_driver"

import roslib;roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("X_Vel", double_t, 0, "X Vel", 0, -0.037, 0.037)
gen.add("Y_Vel", double_t, 0, "Y Vel",0, -0.037, 0.037)
gen.add("Z_Vel", double_t, 0, "Z Vel", 0, -0.037, 0.037)
gen.add("RX_Vel", double_t, 0, "RX Vel", 0, -1, 1)
gen.add("RY_Vel", double_t, 0, "RY Vel", 0,-1, 1)
gen.add("RZ_Vel", double_t, 0, "RZ Vel", 0, -1, 1)

exit(gen.generate(PACKAGE, "JacoVelocity", "JacoVelocity"))